src/main/java/nextstep/courses/domain/SessionUsers.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.exception.CannotRegisterException;
+import nextstep.users.domain.NsUser;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SessionUsers {
+    private final Set<NsUser> users;
+    private final Capacity capacity;
+
+    public SessionUsers(int capacity) {
+        this.capacity = new Capacity(capacity);
+        this.users = new HashSet<>();
+    }
+
+
+    public synchronized void updateCapacity(int value) {
```

Comment:
```suggestion
    public void updateCapacity(int value) {
```

synchronized가 있어야 하는 이유는?
없어도 되지 않을까?